concordium-sdk/src/main/java/com/concordium/sdk/ClientV2MapperExtensions.java

```diff
@@ -874,13 +873,13 @@ static SendBlockItemRequest to(CredentialDeploymentTransaction credentialDeploym
         TransactionTime time = to(credentialDeploymentTransaction.getExpiry().getValue());
 
         return SendBlockItemRequest.newBuilder()
-            .setCredentialDeployment(
-                CredentialDeployment.newBuilder()
-                    .setMessageExpiry(time)
-                    .setRawPayload(ByteString.copyFrom(credentialDeploymentTransaction.getPayloadBytes()))
-                    .build()
+                .setCredentialDeployment(
+                        CredentialDeployment.newBuilder()
+                                .setMessageExpiry(time)
+                                .setRawPayload(ByteString.copyFrom(credentialDeploymentTransaction.getPayloadBytes()))
+                                .build()
                 )
-            .build();
+                .build();
```

Comment:
This does not look like a relevant change.

Reply:
Not sure why it's been reformatted. Changed it back.. for now..